layouts/shortcodes/pages/compatibility-products.html

```diff
@@ -28,7 +28,7 @@ <h2 class="text-center heading-line" id="{{ .title | urlize }}"><span style="bac
           <div class="match-height-item-by-row padding-20 flex-column">
             <div class="center-block padding-left-30 padding-right-30 padding-bottom-10 padding-top-10 grid-center" data-mh="{{ $title | urlize }}">
               {{ if .image }}
-              <a href="{{ .download }}"><img class="img img-responsive" src="{{ .image }}" alt="{{ .vendor }}'s {{ .name }} logo"></a>
+              <a href="{{ .download }}"><img width="{{ .image_width }}" class="img img-responsive" src="{{ .image }}" alt="{{ .vendor }}'s {{ .name }} logo"></a>
```

Comment:
This is probably gonna be duplicate with https://github.com/EclipseFdn/events.eclipse.org/pull/118#event-3962834984

Reply:
It's fine, I should be able to resolve the conflict via GitHub web UI. Thanks for the heads up @flora8984461

Reply:
Sounds good. Just FYI: I'm adding the img_width is because they provide me svg, which will not be shown correctly like png, so that I need a width to make the logo shows.

Reply:
Good catch!